lib/generators/avo/resource_generator.rb

```diff
@@ -272,7 +290,12 @@ def generated_fields_template
             options = ""
             field_options[:options].each { |k, v| options += ", #{k}: #{v}" } if field_options[:options].present?
 
-            fields_string += "\n    #{field_string field_name, field_options[:field], options}"
+            fields_string +=
+              if field_options[:field] == "polymorphic"
+                "\n    #{polymorphic_field_string field_name, options}"
+              else
+                "\n    #{field_string field_name, field_options[:field], options}"
+              end
```

Comment:
If we implement https://github.com/avo-hq/avo/pull/3645/files#r1950414758 we can remove `polymorphic_field_string`and reuse `field_string`.